src/Agent/NewRelic/Agent/Core/Agent.cs

```diff
@@ -130,6 +130,11 @@ public ITransaction CreateKafkaTransaction(MessageBrokerDestinationType destinat
         }
 
         public ITransaction CreateTransaction(bool isWeb, string category, string transactionDisplayName, bool doNotTrackAsUnitOfWork, Action wrapperOnCreate)
+        {
+            return CreateTransaction(isWeb, category, transactionDisplayName, doNotTrackAsUnitOfWork, wrapperOnCreate, (TransactionNamePriority)9999);
```

Comment:
Is a hardcoded 9999 the best we can do here? Could we introduce a `TransactionNamePriority.Highest` value in the enum? I see that `UserTransactionName` is `int.MaxValue` - so maybe we want to use `int.MaxValue -1` for this priority?

Reply:
I removed all of this and simplified things a bit.